src/Extensions/Extensions.jl

```diff
@@ -21,10 +21,16 @@ spliced back into the docstring in place of it:
 
     \\!!get(author)
 
-When no field in found in the metadata for the object, the module and package
+When no field is found in the metadata for the object, the module and package
 metadata are searched in turn.
 """
-Formats.metamacro(::META"get", body, mod, obj) =  Cache.findmeta(mod, obj, symbol(body))
+function Formats.metamacro(::META"get", body, mod, obj)
+    try
+        Cache.findmeta(mod, obj, symbol(body))
+    catch
+        "ERROR: No metadata found for ':$(symbol(strip(body)))'"
```

Comment:
Question: do you think we should in this case set a meta to: `ERROR: No metadata found for ':$(symbol(strip(body)))` or leave it as currently: which has than no meta set and raises an KeyError if accessed.

```julia
"!!get(notset)"
get_notset = ()

```

```
> Docile.Cache.getparsed(Sandbox, :get_notset) 

splices back: "ERROR: No metadata found for ':notset'"

> Docile.Cache.getparsed(Sandbox, :get_notset)

ERROR: LoadError: KeyError: notset not found

```

I could not make my mind up on that!!